util/cli/cli.go

```diff
@@ -204,16 +204,21 @@ func setComments(input []byte, comments string) []byte {
 	return []byte(strings.Join(parts, "\n"))
 }
 
+// InteractiveEdit launches an interactive editor
 func InteractiveEdit(filePattern string, data []byte, save func(input []byte) error) {
 	editor := os.Getenv(editorEnv)
 	if editor == "" {
 		editor = defaultEditor
 	}
+	var editorArgs []string
+	if strings.Contains(editor, " ") {
```

Comment:
I don't think you need the guard condition.

Reply:
This does need tests mind you.

Reply:
yeah so it looks like the circle tests are failing here: https://app.circleci.com/jobs/github/argoproj/argo-cd/10188

And when I try to `make cli` locally, I get a similar error. I'm not sure how to resolve it. Also pinged you on slack.